### PR TITLE
Fixes:  #1117, #1146 RecentlyPlayed - Tablet (Landscape & Portrait) (Low-fi)

### LINKED ISSUE #1117: RecentlyPlayed - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/17910ab6-21f3-4c85-9429-bea2da2b3c26/L-Recently-Played-Stories

Implement low-fi UI for **RecentlyPlayed** tablet landscape mode

**Target PR date**: 25 May 2020
**Target completion date**: 29 May 2020

### LINKED ISSUE #1146: RecentlyPlayed - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/7681fce1-4d15-4d57-aed2-21521f08ce5f/P-Recently-Played-Stories

Implement low-fi UI for **RecentlyPlayed** tablet portrait mode

**Target PR date**: 29 May 2020
**Target completion date**: 30 May 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation
- Implemented low-fi UI for RecentlyPlayed fragment landscape & portrait mode
- Fixes #1117 
- Fixes #1146 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt [hunk 1]
```diff
@@ -13,7 +13,8 @@ class RecentlyPlayedActivityPresenter @Inject constructor(private val activity:
     if (getRecentlyPlayedFragment() == null) {
       activity.supportFragmentManager.beginTransaction().add(
         R.id.recently_played_fragment_placeholder,
-        RecentlyPlayedFragment.newInstance(internalProfileId)
+        RecentlyPlayedFragment.newInstance(internalProfileId),
+        RecentlyPlayedFragment.TAG_RECENTLY_PLAYED_FRAGMENT
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt:17]: Nice. Thanks for doing this.

File: app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt [hunk 2]
```diff
@@ -14,6 +14,7 @@ private const val RECENTLY_PLAYED_FRAGMENT_INTERNAL_PROFILE_ID_KEY = "RecentlyPl
 /** Fragment that contains all recently played stories. */
 class RecentlyPlayedFragment : InjectableFragment(), OngoingStoryClickListener {
   companion object {
+    const val TAG_RECENTLY_PLAYED_FRAGMENT = "recently-played-fragment"
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt]: The value should be `TAG_RECENTLY_PLAYED_FRAGMENT`

File: app/src/main/res/layout/recently_played_fragment.xml [hunk 3]
```diff
@@ -47,7 +47,8 @@
         android:overScrollMode="never"
         android:paddingTop="8dp"
         android:paddingBottom="172dp"
-        android:scrollbars="none" />
+        android:scrollbars="none"
+        android:tag="recently_played_recycler_view" />
```
Comment (rt4914) [inline app/src/main/res/layout/recently_played_fragment.xml]: Why do we need this tag?

File: app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt [hunk 4]
```diff
@@ -530,7 +534,11 @@ class RecentlyPlayedFragmentTest {
           2
         )
       )
-      onView(withId(R.id.ongoing_story_recycler_view)).check(hasGridItemCount(2, 2))
+      if (context.resources.getBoolean(R.bool.isTablet)) {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt:537]: Update the test case name:
`testRecentlyPlayedTestActivity_checkSpanForItem2_spanSizeIsTwo OrThree` 

Ditto for above test case.

**Review by rt4914 — APPROVED** [review]
LGTM, Suggested test case renaming.

File: app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt:17]: Welcome!

File: app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt]: Done!

File: app/src/main/res/layout/recently_played_fragment.xml (hunk 3 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout/recently_played_fragment.xml]: It was added in a middle change that was canceled later.  
That is adding robolectric tests for the span count for each configuration, but I have modified the test cases themselves instead.  
I've removed the tag. PTAL

File: app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt (hunk 4 above)
Comment (MohamedMedhat1998) [reply app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt:537]: Done!

File: app/src/main/res/layout/recently_played_fragment.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/recently_played_fragment.xml]: OKay. Thanks.

**Pull request merged**